multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -443,7 +448,11 @@ def process_one_sample(
                         )
                         is_zero_img = False
                         try:
-                            with PIL.Image.open(img_path) as img:
+                            if feature_modalities.get(per_col_name) == IMAGE_BYTEARRAY:
+                                image_feature = BytesIO(img_feature)
```

Comment:
The assumption here seems to be bytesarray will follow the `RBG` channel standard. I think we should default to `RBG` while allowing user to pass different channel sequence

Reply:
We can definitely make this configurable. I might be wrong but it seems that most pretrained models we support are trained on RBG images, @bryanyzhu do you have any suggestions?

Reply:
In my understanding, most pretrained image models are trained on RGB images in `RGB` channel standard. Only opencv uses `RBG` channel as default.

Reply:
Thanks for the info, that's great to know! I've added this to be configurable.